README.md

```diff
@@ -146,12 +146,20 @@ Linux Flatpak build contributed by [Ethan Lee](https://flibitijibibo.com/).
   * [Haxaplax](https://github.com/haxaplax)
 
 ## Auto Tracker
```

Comment:
Looking through this PR: Isn't that section missing Dread?

Reply:
Ah yes it is. Could add later I guess.